commands/new.command.ts

```diff
@@ -32,6 +32,10 @@ export class NewCommand extends AbstractCommand {
         Collection.NESTJS,
       )
       .option('--strict', 'Enables strict mode in TypeScript.', false)
+      .option(
+            '--caseNaming [caseType]',
+            `Casing type for generated elements. Available options: "pascal", "camel", "kebab-or-snake" (default).`,
```

Comment:
I didn't follow what is the _kebab-or-snake_ type :thinking: 

Also,

```suggestion
            `Casing type for generated files. Available options: "pascal", "camel", "kebab-or-snake" (default).`,
```

as this will cover file names only, as discussed in the issue

Reply:
About the `kebab-or-snake` case:
In the original code there is a case normalization [function](https://github.com/nestjs/nest-cli/blob/master/lib/utils/formatting.ts) that is a mix of kebap and snake case. I kept it for historical purposes, but I also introduced separated kebap and snake normalization functions. I would more than willing to remove it.